README.md

```diff
@@ -1,11 +1,85 @@
+    - `Create backup and close`: Same as `Create Backup`, but if running on desktop, will close the Obsidian window. Will not exit Obsidian app on mobile.
+- Remote 
+    - `Push`
+    - `Pull`
+    - `Edit remotes`
+    - `Remove remote`
+    - `Clone an existing remote repo`: Opens dialog that will prompt for URL and authentication to clone a remote repo
+    - `Open file on GitHub`: Open the file view of the current file on GitHub in a browser window. Note: only works on desktop
+    - `Open file history on GitHub`: Open the file history of the current file on GitHub in a browser window. Note: only works on desktop
+- Local
+    - `Initialize a new repo`
+    - `Create new branch`
+    - `Delete branch`
+    - `CAUTION: Delete repository`
+- Source Control View
+    - `Open source control view`: Opens side pane displaying [Source control view](#sidebar-view)
+    - `Edit .gitignore`
+    - `Add file to .gitignore`: Add current file to .gitignore
+
+## Mobile
+
+### Installation
+
+#### Existing Repo
+
+Follow these instructions for setting up an Obsidian Vault on a mobile device that is already backed up in a remote git repository. 
+
+The instructions assume you are using GitHub, but can be extrapolated to other providers.
+
+1. Make sure any outstanding changes on all devices are pushed and reconciled with the remote repo.
+2. Install Obsidian for Android or iOS.
+3. Create a new vault (or point Obsidian to an empty directory). Do NOT select `Store in iCloud` if you are on iOS.
+4. (Recommended for security) [Create a personal access token on GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). Minimal permissions required are "Read access to metadata" and "Read and Write access to code and commit statuses" for the repo you are going to use.
+5. In Obsidian settings, enable community plugins. Browse plugins to install Obsidian Git.
+6. Enable Obsidian Git (on the same screen)
+7. Go to Options for the Obsidian Git plugin (bottom of main settings page, under Community Plugins section)
+8. Under the "Advanced" section, fill in the username on your git server and your password/personal access token. NOTE: If using a personal access token (recommended), use `__token__` as your username, not your actual GitHub username.
+9. Leave the relative path setting empty (for advanced users only).
+10. Exit plugin settings, open command palette, choose "Obsidian Git: Clone existing remote repo".
+11. Fill in repo URL in the text field and press the repo URL button below it.
+12. Follow instructions to determine the folder to place repo in and whether an .obsidian directory already exits.
+13. Clone should start. Popup notifications (if not disabled) will display the progress. Do not exit until a popup appears requesting that you "Restart Obsidian".
+14. Reenter Obsidian and repeat steps 7 and 8. This is required because the clone overwrites the workspace.
```

Comment:
This can be removed, because the username and password are no longer stored in the `data.json` of the plugin. They are no longer affected by file changes.

Reply:
Done in commit `b55067ffc31ce1c3138bb878c51d70765927f4c4`